src/apis/applicant_api.py

```diff
@@ -14,25 +17,31 @@ def get(self):
         return 'Hello Upload Page!'
 
     def post(self):
-        if 'pdfFile' not in request.files:
+        if 'cover_letter' not in request.files:
             return 'No file part found in POST request.', 400
 
-        pdf_file = request.files['pdfFile']
+        pdf_file = request.files['cover_letter']
```

Comment:
cover_letter is actually correct, because the FormData in the frontend calls it this way. But we can also change it to cv. I need to change something in the frontend anyway.

Reply:
Just checked the frontend and saw that you can´t even just upload and send one file. You need to upload both, cover letter and cv. That should also be changed, so you can send a single cv to the backend and also rename it accordingly. Maybe you can do that too if you are going to work on the frontend?

But after all, if a cv is send to the backend, it should be called cv.

Reply:
Yes you are right i have already changed it in the frontend i just need to to push it. And i will change the name to cv.